programs/drift/src/math/amm_spread.rs

```diff
@@ -95,6 +95,59 @@ pub fn cap_to_max_spread(
     Ok((long_spread, short_spread))
 }
 
+pub fn calculate_long_short_vol_spread(
+    last_oracle_conf_pct: u64,
+    reserve_price: u64,
+    mark_std: u64,
+    oracle_std: u64,
+    long_inten: u64,
+    short_inten: u64,
+    volume_24h: u64,
+) -> DriftResult<(u64, u64)> {
+    // 1.6 * std
+    let market_avg_std_pct = oracle_std
+        .safe_add(mark_std)?
+        .safe_mul(PERCENTAGE_PRECISION_U64)?
+        .safe_div(2 * reserve_price)?;
+
+    let vol_spread: u64 = last_oracle_conf_pct.max(market_avg_std_pct / 2);
+    crate::dlog!(vol_spread, market_avg_std_pct);
+
+    let long_vol_spread_factor: u64 = long_inten
```

Comment:
are long_intensity and volume_24h in the same precision? you divide long_intensity by quote_precision when you update it

Reply:
nope thats `long_intensity_count` which is a `u32`, as it was before it wasnt really creating useful data stream so changed it to $1 precision AVERAGE volume (rather than rolling sum) other `ONE_HOUR`

its not currently used in singles, but could be better future alternative than rolling sum long_intensity / 24h volume so wanted to track